doc/source/_static/css/custom.css

```diff
@@ -1,57 +1,102 @@
-/* For Algolia search box
-    * overflow-y: to flow-over horizontally into main content
-    * height: to prevent topbar overlap
-*/
 #site-navigation {
-  overflow-y: visible;
-  height: 100% !important;
+  overflow-y: auto;
+  position: inherit;
+}
+#site-navigation.collapse {
+  margin-left: -186px;
+}
+.topbar {
+  z-index: 1900;
```

Comment:
I think this causes my nit, what's the reasoning for having that? (not against it per se, just want to understand)